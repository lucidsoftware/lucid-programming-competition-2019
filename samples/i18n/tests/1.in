20
elypyjirg
uo
zcvcwbhhtl
hxn
kyudxaqzs
gx
xpgemdxkf
jqzgekyhp
n
bvs
vgxtqtzhu
vterl
hvrtnqigmv
qcarmzyp
ey
rnxrbbjgzy
vcdsacmz
drzxzb
cwxojm
rbqm
