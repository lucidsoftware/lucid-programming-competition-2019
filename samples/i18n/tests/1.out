e7g
uo
z8l
h1n
k7s
gx
x7f
j7p
n
b1s
v7u
v3l
h8v
q6p
ey
r8y
v6z
d4b
c4m
r2m
